.circleci/config.yml

```diff
@@ -287,6 +296,61 @@ jobs:
           paths:
             - .ccache/
 
+  linux-benchmarks-conbench:
+    executor: build
+    steps:
+      - checkout
+      - run:
+          name: "Update submodules"
+          command: |
+            git submodule sync --recursive
+            git submodule update --init --recursive
+      - run:
+          name: "Calculate merge-base date for CCache"
+          command: |
+            git show -s --format=%cd --date="format:%Y%m%d" $(git merge-base origin/main HEAD) | tee merge-base-date
+      - restore_cache:
+          name: "Restore CCache cache"
+          keys:
+            - velox-ccache-release-{{ arch }}-{{ checksum "merge-base-date" }}
+      - run:
+          name: "Build Benchmarks"
+          command: |
+            mkdir -p .ccache
+            export CCACHE_DIR=$(realpath .ccache)
+            ccache -sz -M 5Gi
+            source /opt/rh/gcc-toolset-9/enable
+            make benchmarks-basic-build NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8
+            ccache -s
+      - run:
+          name: "Install conbench and veloxbench"
+          command: |
+            # upgrade python to 3.8 for this job
+            yum install -y python38
+            pip3.8 install conbench
+            pip3.8 install scripts/veloxbench
+      - run:
+          name: "Run Benchmarks"
+          command: |
+            # get to the directory with benchmarks
+            cd scripts/veloxbench/veloxbench
+
+            # setup .conbench file
```

Comment:
should we have this file baked by default into our docker image?

Reply:
We could, though we probably should keep the secret env vars configured here and not in the image so that they aren't available for anyone who downloads the image.